.github/workflows/notebooks.yml

```diff
@@ -16,15 +20,29 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.10
+
+    - name: Set up Python
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
+
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip --quiet install --upgrade .[complete]
-        python -m pip list
+        python -m pip install --upgrade .[test]
+
+    - name: List installed Python packages
+      run: python -m pip list
+
+    - name: REMOVE test a failing test by self
+      run: |
+        pytest --override-ini filterwarnings= tests/test_notebooks.py -k test_xml_importexport
+
     - name: Test example notebooks
       run: |
-        pytest tests/test_notebooks.py
+        # Override the ini option for filterwarnings with an empty list to disable error
+        # on filterwarnings as testing for notebooks to run with the latest API, not if
+        # Jupyter infrastructure is warning free.
+        # Though still show warnings by setting warning control to 'default'.
+        # export PYTHONWARNINGS='default'
```

Comment:
I don't understand why having `PYTHONWARNINGS='default'` doesn't work, so maybe this will become an Issue unto itself and then we can just move on here.

Reply:
I've just summarized things in Issue #1840.